modules/puppet/spec/classes/puppet_spec.rb

```diff
@@ -18,7 +18,7 @@
     }}
 
     it do
-      is_expected.to contain_file('/etc/puppet/puppet.conf').with_content(/environmentpath = \/usr\/share\/puppet\/production\/current\/environments/)
+      is_expected.to contain_file('/etc/puppet/puppet.conf').with_content(/environmentpath = \/usr\/share\/puppet\/rp_env\/current\/environments/)
```

Comment:
This looks like it's a variable - where does it come from? The `puppet.conf.erb` file contains `<%= scope.lookupvar('::environment') %>`.

Reply:
`rp_env` is what rspec-puppet sets `$::environment` to. I'll add a comment to clarify that.